src/services/reCaptcha.js

```diff
@@ -14,11 +14,24 @@ class ReCaptcha {
 
   async askGoogle() {
     const postUrl = `${this.url}?secret=${this.secret}&response=${this.userToken}`;
+    const postData = {};
+    const timeoutMs = 5 * 1000;
```

Comment:
why is 5 seconds the right amount of time to wait for a timeout? i see the default before was no timeout? just wondering if we should set this to a lower number, like 2 seconds, since we're retrying 3x. up to 15 seconds to wait for google recaptcha to respond seems like a bit longer than helpful? not sure.

Reply:
That's a good point. Looking at App Insights, the 99th percentile in the last 24 hours for the recaptcha call is 128ms. I think a lower number like 2s should be fine. I'll change it to 2s and then land.

![Annotation 2020-07-17 144600](https://user-images.githubusercontent.com/175378/87833126-446fec00-c83c-11ea-903a-357813acb5ad.png)

Reply:
oh that's great data to base it on-- thanks!